internal/parse/parse_internal_test.go

```diff
@@ -1,61 +0,0 @@
-package parse
-
-import (
-	"log"
-	"testing"
-)
-
-type parseHelperTest struct {
-	bytef    func(byte) bool
-	stringf  func(string) bool
-	stringf0 func() bool
-	result   []bool
-	input    string
-	data     []string
-}
-
-func TestRunTable(t *testing.T) {
```

Comment:
Why was this dropped?  Is it coming back in a different shape?

Reply:
It is! In a different file - see #13